src/test/java/com/slack/kaldb/config/KaldbConfigTest.java

```diff
@@ -135,6 +136,53 @@ public void testParseKaldbConfigFile() throws IOException {
     assertThat(indexerConfig.getDataDirectory()).isEqualTo("/tmp");
   }
 
+  @Test
```

Comment:
Can we add one more test that sets an env var and then asserts if the value is set correctly?

This example ( https://stackoverflow.com/questions/8168884/how-to-test-code-dependent-on-environment-variables-using-junit ) ensures that we unset it after the test

I guess we can do it ourselves in try/finally block

Reply:
Per our Slack conversation we'll leave this test out for now. We can revisit this if we decide on a test framework like Mockito or similar. I've added a comment to the test line in question explaining this decision as well.